
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

revolute1_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: revolute1
  pid: {p: 10.0, i: 0.0, d: 0.1}

revolute2_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: revolute2
  pid: {p: 10.0, i: 0.0, d: 0.1}

arm_trajectory_controller:
  type: effort_controllers/JointTrajectoryController
  joints:
    - revolute5
    - revolute6
  pid:
    revolute5: {p: 25.0, i: 0.0, d: 0.5}
    revolute6: {p: 25.0, i: 0.0, d: 0.5}
  constraints:
    goal_time: 2.0
    stopped_velocity_tolerance: 0.02
